Merging Development 8.3 : Moving Changes from Development 7.67 to Development 8.3 Environment

omkarchoudhari opened the pull request:
## Description

> No Screenshots : 9th October 2024

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

poojagurav21 committed:
committed changes for Assistance & developer mode

poojagurav21 committed:
Changes for save button disable on required field

1.Disable save button
2.Validation message

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug/654/Assistance/RequiredMsgForEmail

suchithravipin1991 committed:
Pagination issues resolved

Design System > Pagination Issues

On hover grid dropdown hides the number

suchithravipin1991 committed:
Counter Input issue resolved

Design System > Action Issues
#678
3.Cannot manually enter the value & while binding the value back it is not get assigned.

suraj-c committed:
Commit changes Side navigation is not as per the application

suraj-c committed:
Commit changes for remove unwanted code

prakashmhaske committed:
commit changes for  Multiple selects are not working.

suchithravipin1991 committed:
Updated counter.tsx

suraj-c committed:
Commit changes for Implement different types of Alerts

suraj-c committed:
Commit changes for elements input tooltip the eye icon is overlapping

rohit9096 committed:
develop new badges

rohit9096 committed:
Merge remote-tracking branch 'origin/development' into users/rohit/605-Design-System-Implement-different-Badge

poojagurav21 committed:
changes removed for css

Ketan-Khalate committed:
Merge branch 'development' into user/pooja/bug/654/Assistance/RequiredMsgForEmail

Ketan-Khalate committed:
Merge branch 'development' into users/suchithra.aa/bugs/designSystem/pagination-issues

Pundit-Assistant committed:
Updated LICENSE to MIT

poojagurav21 committed:
committed changes for Storybook > Component >Login > Default > Tablet view chevron is not visible proper.  #420

poojagurav21 committed:
committed changes for Bug 667: Storybook > Component > Small and Large Mabile view save and cancel button is alignment issue all component.

poojagurav21 committed:
committed changes for Bug 661 : Misplaced Position of Setting Icon on Sidebar

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug/654/Assistance/RequiredMsgForEmail

poojagurav21 committed:
updated rds-developer-mode for disable button issue

suchithravipin1991 committed:
Storybook >Charts>Scatter Chart>Docs> Scatter Chart icon does not display proper.

Storybook >Charts>Scatter Chart>Docs> Scatter Chart icon does not display proper is resolved

poojagurav21 committed:
Committed changes for Design System > Disable Save button till all mandatory fields have the value

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug/688/Component/DisableSaveButton

poojagurav21 committed:
committed changes for  Design System > Components > Get Assistance > Default > Fill the required field. NEXT button is not enabled.

poojagurav21 committed:
committed changes for Storybook > Component >Feeds > Month text is not as per wireframe.  #731

poojagurav21 committed:
changes for Long Feed and Review component

poojagurav21 committed:
committed changes for date format

prakashmhaske committed:
commit changes for popover issue

poojagurav21 committed:
committed changes for  Storybook > Component > Admin Dashboard> In small mobile view Card Revenue, Sales and To do List table is not display proper.

suchithravipin1991 committed:
#736 - Design system pie chart space issue resolved

#736 - Design system pie chart space issue resolved

prakashmhaske committed:
commit changes for application shells  header  add layout here is not display full screen

suchithravipin1991 committed:
#458: Alertpopup  issues resolved

#458: Alertpopup  issues resolved

suchithravipin1991 committed:
#476 Scrollbar Alignment issue is not present

Scroll bar alignment be proper.
Search input remove unused characters.
Label is Ci/Cd instead of CICD.   All these points working properly

hemalatagaikwad committed:
Merge pull request #751 from Wai-Technologies/users/suchithra.aa/designsystems/elements-issue

Resolved Bug 746 : Storybook >Design System > Scroll bar alignment issue.

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/731/Component-Feeds

hemalatagaikwad committed:
Merge pull request #734 from Wai-Technologies/user/pooja/bug/731/Component-Feeds

Resolved Bug 731 : Storybook > Component > Feeds & Reviews > Month text is not as per wireframe

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/715/Component-GetAssistance

hemalatagaikwad committed:
Merge pull request #730 from Wai-Technologies/user/pooja/bug/715/Component-GetAssistance

Resolved Bug 715 :  Design System > Components > Get Assistance > Default > Fill the required field. NEXT button is not enabled.

prakashmhaske committed:
commit changes for bug 733

suchithravipin1991 committed:
Merge branch 'development' into users/suchithra.aa/designsystems/alertissue

suchithravipin1991 committed:
buttons added

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/654/Assistance/RequiredMsgForEmail

suchithravipin1991 committed:
Bug resolved 753: Card enhancement

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/420/Login

hemalatagaikwad committed:
Merge pull request #720 from Wai-Technologies/user/pooja/bug/420/Login

Resolved bug 420: Storybook > Component >Login > Default > Tablet view chevron is not visible proper.

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/661/Sidebar-SettingIcon

poojagurav21 committed:
carousel 4 styles implemented

hemalatagaikwad committed:
Merge pull request #722 from Wai-Technologies/user/pooja/bug/661/Sidebar-SettingIcon

Resolved Bug 661 : Local environment > Design system > side nav bar > setting icon> misplaced the position of setting icon

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/667/Small-LargeMobileViewButtonIssue

poojagurav21 committed:
select list placeholder changes

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug/654/Assistance/RequiredMsgForEmail

hemalatagaikwad committed:
Merge pull request #721 from Wai-Technologies/user/pooja/bug/667/Small-LargeMobileViewButtonIssue

Resolved Bug 667 : Storybook > Component > Small and Large Mabile view save and cancel button is alignment issue all component

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/654/Assistance/RequiredMsgForEmail

hemalatagaikwad committed:
Merge pull request #687 from Wai-Technologies/user/pooja/bug/654/Assistance/RequiredMsgForEmail

Resolved Bug 172, 434, 654 and 658 : Missing required validation message and save button disable based on required field

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/688/Component/DisableSaveButton

poojagurav21 committed:
committed changes for url format

poojagurav21 committed:
Merge branch 'user/pooja/bug/688/Component/DisableSaveButton' of https://github.com/Wai-Technologies/raaghu-react into user/pooja/bug/688/Component/DisableSaveButton

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/735/Component-AdminDashboard

rahulp34 committed:
Commited changes

rahulp34 committed:
commit merge

poojagurav21 committed:
Url validation

rahulp34 committed:
commited changes

hemalatagaikwad committed:
Merge pull request #763 from Wai-Technologies/users/rahul/react/708/Compare-all-storybooks-elements-components-charts-app-shell-layout

Completed Task 708 : Design System > Compare all storybook's elements, components ,charts, app shell and layout with design link and fix the differences

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/688/Component/DisableSaveButton

hemalatagaikwad committed:
Merge pull request #729 from Wai-Technologies/user/pooja/bug/688/Component/DisableSaveButton

Resolved Bug 688: Design System > Disable Save button till all mandatory fields have the value

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/735/Component-AdminDashboard

suchithravipin1991 committed:
cards modified

suchithravipin1991 committed:
Merge branch 'development' into users/suchithra.aa/designsytems/Card-issue

hemalatagaikwad committed:
Merge pull request #741 from Wai-Technologies/user/pooja/bug/735/Component-AdminDashboard

Resolved Bug 735: Storybook > Component > Admin Dashboard> In small mobile view Card Revenue, Sales and To do List table is not display proper.

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/designsytems/Card-issue

hemalatagaikwad committed:
Merge pull request #761 from Wai-Technologies/users/suchithra.aa/designsytems/Card-issue

Resolved  Bug 753: Design System > Elements > Card : Minor Issues

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/designsystems/alertissue

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/bug-storybook-scatter-chart-icon-issue

hemalatagaikwad committed:
Merge pull request #728 from Wai-Technologies/users/suchithra.aa/bug-storybook-scatter-chart-icon-issue

Resolved Bug 522 : Storybook > Charts > Scatter Chart > Docs > Scatter Chart icon does not display issue is resolved

poojagurav21 committed:
Changes for carousel style are implemented

poojagurav21 committed:
Merge branch 'development' into user/pooja/bug/754/Carousel-MinorIssue

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/raaghu-design-systems/counter-input-issue

poojagurav21 committed:
Changes to remove the Default story from the carousel

hemalatagaikwad committed:
Merge pull request #698 from Wai-Technologies/users/suchithra.aa/raaghu-design-systems/counter-input-issue

Resolved Bug 678 : Counter Input issue resolved > Cannot manually enter the value & while binding the value back it is not get assigned

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/designsytems-piechart-space-issue

hemalatagaikwad committed:
Merge pull request #742 from Wai-Technologies/users/suchithra.aa/designsytems-piechart-space-issue

Resolved Bug 736 : Design System > Pie Chart space issue resolved

suchithravipin1991 committed:
Resolved Bug 722: Input type number added

hemalatagaikwad committed:
Merge branch 'development' into users/suraj/bug-662-elements-input-tooltip-the-eye-icon-is-overlapping

hemalatagaikwad committed:
Merge pull request #712 from Wai-Technologies/users/suraj/bug-662-elements-input-tooltip-the-eye-icon-is-overlapping

Task 662: Design system > Elements > Input > Tooltip > The eye icon is overlapping

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/bug-722-input-type-number-is-not-present

hemalatagaikwad committed:
Merge pull request #783 from Wai-Technologies/users/suchithra.aa/bug-722-input-type-number-is-not-present

Resolved Bug 722 : Design System > Input type number is not present at all in raaghu element

poojagurav21 committed:
Enhancement for  Design System > Elements > Card Enhancements #782

poojagurav21 committed:
Merge branch 'development' into user/pooja/Enhancement/782/Card-Enhancement

prakashmhaske committed:
Merge branch 'development' into users/prakash/select-list-multiple-selects-are-not-working

suchithravipin1991 committed:
Task Done Task ID:785- Enhancement in videoplayer has done

suchithravipin1991 committed:
updated

suchithravipin1991 committed:
Bug Resolved 655: Billing Address label issue resolved

suchithravipin1991 committed:
button updated also  included with this branch

hemalatagaikwad committed:
Merge pull request #799 from Wai-Technologies/users/suchithra.aa/bug-655-billingaddress-label-ussue

Bug Resolved 655: Billing Address label issue resolved

DeepashriDatar committed:
Added changes in Checkbox element (Enhancement)

hemalatagaikwad committed:
Merge pull request #800 from Wai-Technologies/users/deepashri/task/755/Elements-Checkbox-enhancements

Completed Task 755 : Design System > Elements > Checkbox: Few enhancements

hemalatagaikwad committed:
Merge branch 'development' into user/pooja/bug/754/Carousel-MinorIssue

hemalatagaikwad committed:
Merge pull request #779 from Wai-Technologies/user/pooja/bug/754/Carousel-MinorIssue

Enhancement 754 : Design System > Elements > Carousel : Few Enhancements

poojagurav21 committed:
border color and color variant property changes

poojagurav21 committed:
Merge branch 'development' into user/pooja/Enhancement/782/Card-Enhancement

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/Task/785-enhancement-in-videoplayer

hemalatagaikwad committed:
Merge pull request #797 from Wai-Technologies/users/suchithra.aa/Task/785-enhancement-in-videoplayer

Completed Task 785 : Design System > Elements > Enhancements in Video Player

hemalatagaikwad committed:
Merge pull request #791 from Wai-Technologies/user/pooja/Enhancement/782/Card-Enhancement

Enhancement for 782: Design System > Elements > Card Enhancements

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/bugs/designSystem/pagination-issues

hemalatagaikwad committed:
Merge pull request #697 from Wai-Technologies/users/suchithra.aa/bugs/designSystem/pagination-issues

Resolved Bug 676 : Pagination Issues Resolved

hemalatagaikwad committed:
Merge branch 'development' into users/suchithra.aa/designsystems/alertissue

hemalatagaikwad committed:
Merge pull request #748 from Wai-Technologies/users/suchithra.aa/designsystems/alertissue

Resolved Bug 458 :  Storybook > Alert Popup / Docs : Unable to see the fields properly

hemalatagaikwad committed:
Merge branch 'development' into users/prakash/bug-716-popover-unable-see-popover-clearly-

hemalatagaikwad committed:
Merge pull request #738 from Wai-Technologies/users/prakash/bug-716-popover-unable-see-popover-clearly-

Resolved Bug 716 : Design System > Elements Popover Docs Vertical Scrollbar Is Missing so Unable To See Popover Clearly

hemalatagaikwad committed:
Merge branch 'development' into users/prakash/select-list-multiple-selects-are-not-working

hemalatagaikwad committed:
Merge pull request #705 from Wai-Technologies/users/prakash/select-list-multiple-selects-are-not-working

Resolved Bug 699 : Design System > Elements > Select List > Multiple > Multiple Selects are not working

hemalatagaikwad committed:
Merge branch 'development' into users/prakash/bug/695-application-shells--header-add-layout-here-is-not-display-full-screen

hemalatagaikwad committed:
Merge pull request #745 from Wai-Technologies/users/prakash/bug/695-application-shells--header-add-layout-here-is-not-display-full-screen

Resolved Bug 695 : Storybook > Application Shells > Header > Add Layout Here is Not Display Full Screen

hemalatagaikwad committed:
Merge branch 'development' into users/suraj/side-navigation-is-not-as-per-the-application

hemalatagaikwad committed:
Merge pull request #702 from Wai-Technologies/users/suraj/side-navigation-is-not-as-per-the-application

Completed Task 519 : Storybook > Elements > Side Navigation > Side Navigation Is not as per the application

hemalatagaikwad committed:
Merge pull request #710 from Wai-Technologies/users/suraj/task-590-implement-different-types-of-alerts

Task 590 : Design System > Implement Different Types Of Alerts

hemalatagaikwad committed:
Merge branch 'development' into users/rohit/605-Design-System-Implement-different-Badge

hemalatagaikwad committed:
Merge pull request #714 from Wai-Technologies/users/rohit/605-Design-System-Implement-different-Badge

Completed Task 605 : Design System > Implement different Badge

rahulp34 committed:
Commited changes for alert and badge

rahulp34 committed:
Merge remote-tracking branch 'origin/development' into users/rahul/react/708/Compare-all-storybooks-elements-components-charts-app-shell-layout

hemalatagaikwad committed:
Merge pull request #817 from Wai-Technologies/users/rahul/react/708/Compare-all-storybooks-elements-components-charts-app-shell-layout

Users/rahul/react/708/compare all storybooks elements components charts app shell layout

poojagurav21 committed:
changes for removing the error msg

hemalatagaikwad committed:
Merge pull request #818 from Wai-Technologies/user/pooja/bug/812/AreaChartErrorMsg

Resolved Bug 812 : Storybook > Charts > Area Chart > Show error message

rahulp34 committed:
commit changes for font, font-size, Headings, border-radius,

hemalatagaikwad committed:
Merge pull request #824 from Wai-Technologies/users/rahul/Bug/822/Design-System-Create-common-variables-for-bootstrap-utilities

Completed Task 897 : Create common variables for bootstrap utilities.

kaushikgokhale committed:
Merge branch 'development' into users/kaushik/changes-for-license-to-MIT

omkarchoudhari committed:
Merge pull request #718 from Wai-Technologies/users/kaushik/changes-for-license-to-MIT

Completed Task 700 : Updated LICENSE to MIT for Design System Open Source Repo

kaushikgokhale committed:
Merge branch 'development' into users/kaushik/changes-with-respect-to-8.3.3

hemalatagaikwad committed:
resolved build errors
